src/test/java/org/broadinstitute/hellbender/utils/tsv/TableWriterUnitTest.java

```diff
@@ -253,8 +255,12 @@ protected void composeLine(final TableReaderUnitTest.TestTuple record, final Dat
         };
     }
 
-    private List<String> outputLines(File testFile) throws FileNotFoundException {
-        final BufferedReader bufferedReader = new BufferedReader(new FileReader(testFile));
-        return bufferedReader.lines().collect(Collectors.toList());
+    private List<String> outputLines(Path testPath) throws FileNotFoundException {
+
+        try (BufferedReader bufferedReader = Files.newBufferedReader(testPath)) {
+            return bufferedReader.lines().collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new FileNotFoundException(e.getMessage());
```

Comment:
Same comment here as above regarding narrowing the exception: drop the try-catch and wrapping, and just declare `throws IOException`. Most of the callers already declare that.

Reply:
Done. Looks like it was all callers because this compiled without complaint.